### PR TITLE
Update openria nuget to 5.0.0

### PR DESCRIPTION
**Pull request by niklas-holma**
Update openria nuget to 5.0.0

We will have to wait until the 5.0 release of FluentMetadata nuget pagage is created until this can build.

### CODE REVIEW

File: OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
-
-  <ItemGroup>
-    <ProjectReference Include="..\OpenRiaServices.M2M.LinkTable\OpenRiaServices.M2M.LinkTable.csproj" />
-  </ItemGroup>
-
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net472</TargetFrameworks>
+    <Description>Server part of M2M support for OpenRiaServices. It includes Code generator and Fluent configuration extensions for OpenServices.M2M.</Description>
```
Comment (Daniel-Svensson) [inline OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj]: Why does the whole file show up as a diff? Cannot se on mobile

**Review by Daniel-Svensson — APPROVED** [review]
Assuming only the nuget version of dependencies is updated go ahead and merge, crate a tag (github release with notes) and then deploy the ci build of the tag in devops

File: OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj (hunk 1 above)
Comment (niklas-holma) [reply OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj]: seems to be some kind of whitespace diff

Comment (niklas-holma) [reply OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj]: After further investigation it seems like the original file only used linefeed endings
I assume we want windows standard CR LF ?

Comment (Daniel-Svensson) [reply OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj]: Cr lf should be fine, git should convert it to lf on check in given standard gitattribute file

You might want to check that there is a default gitattribute file in the root such as https://github.com/dotnet/core/blob/master/.gitattributes

Comment (niklas-holma) [reply OpenRiaServices.M2M/OpenRiaServices.M2M.Server.csproj]: Done

**Pull request merged**